SmartWeights/SmartWeights/Workout/StartWorkout.swift

```diff
@@ -438,9 +463,10 @@ class WorkoutViewModel: ObservableObject {
     
     func playSound() {
         do {
-
-            try AVAudioSession.sharedInstance().setCategory(.playAndRecord, mode: .default, options: [.defaultToSpeaker])
-            try AVAudioSession.sharedInstance().setMode(.default)
+            // try AVAudioSession.sharedInstance().setCategory(.playback, mode: .default, options: [])
```

Comment:
If these are unnecessary, just remove em

Reply:
mb i thought i removed it already